test/REGRESSIONS

```diff
@@ -686,6 +679,668 @@ Reviewed 2015-01-23
+cray-module-correctness-cray-xc-slurm-none-gnu
+Inherits 'cray-module-correctness-cray-xc-none-gnu'
+and 'cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+===================================================
+
+
+============================================================
+cray-module-correctness-cray-xc-slurm-gasnet-aries-cray
+Inherits 'cray-module-correctness-cray-xc-gasnet-aries-cray'
+and ''cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+============================================================
+
+
+===========================================================
+cray-module-correctness-cray-xc-slurm-gasnet-aries-pgi
+Inherits 'cray-module-correctness-cray-xc-gasnet-aries-pgi'
+and ''cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+===========================================================
+
+
+=============================================================
+cray-module-correctness-cray-xc-slurm-gasnet-aries-intel
+Inherits 'cray-module-correctness-cray-xc-gasnet-aries-intel'
+and ''cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+=============================================================
+
+
+===========================================================
+cray-module-correctness-cray-xc-slurm-gasnet-aries-gnu
+Inherits 'cray-module-correctness-cray-xc-gasnet-aries-gnu'
+and ''cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+===========================================================
+
+
+==========================================================
+cray-module-correctness-cray-xc-slurm-gasnet-mpi-cray
+Inherits 'cray-module-correctness-cray-xc-gasnet-mpi-cray'
+and 'cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+==========================================================
+
+
+=========================================================
+cray-module-correctness-cray-xc-slurm-gasnet-mpi-pgi
+Inherits 'cray-module-correctness-cray-xc-gasnet-mpi-pgi'
+and 'cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+=========================================================
+
+
+===========================================================
+cray-module-correctness-cray-xc-slurm-gasnet-mpi-intel
+Inherits 'cray-module-correctness-cray-xc-gasnet-mpi-intel'
+and 'cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+===========================================================
+
+
+=========================================================
+cray-module-correctness-cray-xc-slurm-gasnet-mpi-gnu
+Inherits 'cray-module-correctness-cray-xc-gasnet-mpi-gnu'
+and 'cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+=========================================================
+
+
+=====================================================
+cray-module-correctness-cray-xc-slurm-ugni-intel
+Inherits 'cray-module-correctness-cray-xc-ugni-intel'
+and 'cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+=====================================================
+
```

Comment:
My organization in this way was a reaction to more overlap than I had expected.  I'll review it when I get through today's mail.